### PR TITLE
TechSmith.Snagit.2023 version 23.0.1.23930

### PR DESCRIPTION
**Pull request by ajsween**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90138)

### CODE REVIEW

**Comment by ajsween** [comment]
> Hello @ajsween,
> 
> The package manager bot determined there was an issue with some of the URLs included in the manifest file. Please check the pull request for more details and make sure the urls are correct.
> 
> Template: msftbot/validationError/urls/smartScreen

I've double checked the URLs. They all work. My guess is that the site is blocking access to the release notes based on user-agent used by the validation pipeline.

**Comment by ajsween** [comment]
Not sure what to do here. It appears the support urls at TechSmith blocking crawling.

File: manifests/t/TechSmith/Snagit/2023/23.0.1.23930/TechSmith.Snagit.2023.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: TechSmith.Snagit.2023
+PackageVersion: 23.0.1.23930
+PackageLocale: en-US
+Publisher: TechSmith Corporation
+PublisherUrl: https://www.techsmith.com/screen-capture.html
+PublisherSupportUrl: https://support.techsmith.com/hc/en-us/categories/200174548-Snagit-Windows-Technical-Issues
```
Comment (mdanish-kh) [inline manifests/t/TechSmith/Snagit/2023/23.0.1.23930/TechSmith.Snagit.2023.locale.en-US.yaml:9]:
```suggestion
# PublisherSupportUrl:
```

File: manifests/t/TechSmith/Snagit/2023/23.0.1.23930/TechSmith.Snagit.2023.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    Fixed a crash that could occur when overwriting changes to certain file formats with Save As.
+    Fixed a crash when switching categories in the Library and then immediately attempting to share.
+    Fixed a crash that could occur when dragging items in the Quick Styles pane.
+    Fixed a crash that sometimes occurred after connecting or disconnecting webcam or microphone devices.
+    Fixed an issue where the Share Link button and other sharing options didn't always re-enable properly.
+    Fixed an issue where the Screen Draw button sometimes didn’t appear after pausing fullscreen recordings.
+    Fixed an issue where Screen Draw objects didn't always appear on top of the webcam during video recordings.
+    Fixed text wrapping issue in the Finish Sharing dialog.
+    Other bug fixes and performance improvements.
+ReleaseNotesUrl: https://support.techsmith.com/hc/en-us/articles/115006435067-Snagit-Windows-Version-History
```
Comment (mdanish-kh) [inline manifests/t/TechSmith/Snagit/2023/23.0.1.23930/TechSmith.Snagit.2023.locale.en-US.yaml:48]:
```suggestion
# ReleaseNotesUrl:
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
As these fields are not mandatory, the resolution would be to get rid of them to pass the CI Pipelines.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**